tests/pr_test.go

```diff
@@ -32,10 +34,10 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 		TerraformDir:                  dir,
 		Prefix:                        prefix,
 		ResourceGroup:                 resourceGroup,
+		Region:                        region, // Not all regions provide cross region support so value must be hardcoded https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-service-availability.
 		CloudInfoService:              sharedInfoSvc,
 		ExcludeActivityTrackerRegions: true,
 	})
-
```

Comment:
ExcludeActivityTrackerRegions can now be removed, we wont be deploying activity tracker anymore. It also means the `TestMain` function can be removed too